src/main/webapp/app/lecture/lecture-update.component.html

```diff
@@ -1,131 +1,132 @@
                             <label class="form-control-label" jhiTranslate="artemisApp.lecture.course" for="field_course"></label>
-                            <div class="d-flex">
-                                <input id="field_course" disabled type="text" class="form-control" name="course" [(ngModel)]="lecture().course!.title" />
-                            </div>
+                            @if (lecture().course?.title) {
+                                <div class="d-flex">
+                                    <input id="field_course" disabled type="text" class="form-control" name="course" [(ngModel)]="lecture().course!.title!" />
+                                </div>
+                            }
+                        </div>
+                    }
+                    <jhi-lecture-title-channel-name [lecture]="lecture()" />
+                    @if (lecture()) {
+                        <div class="form-group" id="field_description">
+                            <label class="form-control-label" jhiTranslate="artemisApp.lecture.description" for="field_description"></label>
+                            <jhi-markdown-editor-monaco
+                                class="markdown-editor"
+                                [domainActions]="domainActionsDescription"
+                                [(markdown)]="lecture().description"
+                                (markdownChange)="updateIsChangesMadeToTitleOrPeriodSection()"
+                            />
                         </div>
                     }
                 </div>
-                <div class="py-2">
-                    <input class="form-check-input" type="checkbox" name="processUnit" (change)="onSelectProcessUnit()" [disabled]="editForm.form.invalid || isSaving" />
-                    <span class="px-1" jhiTranslate="artemisApp.attachmentUnit.createAttachmentUnits.processAutomatically"></span>
-                    <fa-icon [icon]="faQuestionCircle" [ngbTooltip]="'artemisApp.attachmentUnit.createAttachmentUnits.lectureFileToolTip' | artemisTranslate" />
+                <jhi-lecture-update-period [lecture]="lecture()" [validateDatesFunction]="onDatesValuesChanged" />
+                <div class="flex-grow-1 d-flex justify-content-end">
+                    <button type="button" id="cancel-save" class="btn btn-secondary" (click)="previousState()">
+                        <fa-icon [icon]="faBan" />&nbsp;<span jhiTranslate="entity.action.cancel"></span>
+                    </button>
+                    <button
+                        type="submit"
+                        id="save-entity"
+                        class="btn btn-primary ms-2"
+                        [disabled]="editForm.form.invalid || isSaving || processUnitMode || !isChangeMadeToTitleOrPeriodSection || !areSectionsValid()"
+                    >
+                        <fa-icon [icon]="faSave" />&nbsp;<span jhiTranslate="entity.action.save"></span>
```

Comment:
Here is another `&nbsp;`

Reply:
I was not aware that I have introduced it at multiple places  - I now exchanged all `nbsp` in this file 👍